lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -76,6 +76,17 @@ module GRPC
   extend FluentLogger
 end
 
+require 'strptime'
```

Comment:
This is part of the disabling of the gem, so should probably be after the comment (less chance that it'll accidentally get moved to the top, for example).